Comp2023-Imported/src/main/java/frc/robot/commands/AutoDrivePath.java

```diff
@@ -47,7 +47,7 @@ public void initialize( )
   @Override
   public void execute( )
   {
-    m_swerve.driveWithPathFollowerExecute( );
+    m_swerve.driveWithPathFollowerExecute(m_useInitialPose);

```

Comment:
I'm not sure what the initial pose has to do while we're executing a path follower. The check should only be when the path is initialized, right? Let's review what this is doing and where it came from.